_i18n/en.yml

```diff
@@ -0,0 +1,81 @@
+---
+global:
+  brazilian portuguese: Português
+  english: English
+header:
+  stories: Stories
+  explore: Explore
+  about: About
+support:
+  fund_us: Support the project
+home:
+  about: Artificial Intelligence for social control of public administration
+  about_rosie: Operação Serenata de Amor created a Robot - An artificial intelligence capable of analysing each claim for reimbursement from congresspeople and of identifying it’s illegal probability.
+  last_histories: Last stories
+  loading: Loading…
+  more_posts: More stories
+footer:
+  about: about
+  github: Serenata on github
+  contact: contact
+  partners: partners
+stories:
+  published: Stories
+  loading: Loading…
+explore:
+  reports: Reports
+  about_reports: Rosie, our Robot, in numbers
+  tweet_about: Tweet about this figure
+  chamber_reports: Cases reported to the Chamber of Deputies
+  suspicions: Different congresspeople reported for suspicious reimbursements
+  questionated: thousand Brazilian Reals questioned in suspicious expenses
+  paragraph_one: This amount corresponds to 371% of what we raised through our crowdfunding campaign (~US$ 25,000.00)
+  paragraph_two: Want to know how the report system works?
+  more: You can read more in this post (in Portuguese)
+about:
+  title: Serenata de amor operation
```

Comment:
Serenata de amor operation have 2  matches
Love serenade also have 2 matches. 

We have to choose how we gonna call the project.